fusor-ember-cli/app/styles/custom.scss

```diff
@@ -940,3 +940,34 @@ span.hanging-indent{
   vertical-align: top;
   margin-bottom: 0px !important; /* overides form-group default */
 }
+
+.deployments-table {
+  table-layout:fixed;
+  width: 100%;
+}
+.deployments-table>thead:first-child>tr:first-child>th:first-child
+{
+    width: 40%;
+}
+.deployments-table> tbody > tr > td:first-child {
+    width: 40%;
+}
+
+.pagination {
+  display: table-cell;
+  vertical-align: bottom;
+  float: none;
+}
+
+.darkgrey, .darkgray {
+  color: #606060 !important;
+}
+
+.displaying-entries {
+  margin-left: -20px;
+}
+
+.rhci-title {
+  display: inline-block;
```

Comment:
@cfchase `inline-block` for the "Deployments" title is what caused the new button `float: right` to appear too high.